apps/src/javalab/JavalabConsole.jsx

```diff
@@ -152,7 +153,7 @@ class JavalabConsole extends React.Component {
     } else {
       return (
         <div onClick={this.onLogsClick} style={styles.logs}>
-          {this.renderConsoleLogs(isDarkMode)}
+          {this.renderConsoleLogs(displayTheme === DisplayTheme.DARK)}
```

Comment:
can you also refactor `renderConsoleLogs` to take in `displayTheme` so that function does not have an `isDarkMode` parameter?

Reply:
Yep, done! `isDarkMode` officially no longer in our code base after that :)